docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: doc_page
+---
+
+What to Do When You Have a Firewall
+-----------------------------------
+When you are behind a firewall, the Maven Druid dependencies will not be accessible, as well as the IRC wikipedia channels that feed realtime data into Druid. To workaround those two challenges, you will need to:
```

Comment:
I think we should distinguish IRC being blocked by a firewall vs. nodes that cannot even talk http to the outside world.